docs/environment.yml

```
@@ -10,7 +10,9 @@ dependencies:
 -   sphinx-copybutton
 -   sphinx-design
 -   sphinx-togglebutton
+
 -   pip:
     -   sphinxemoji
     -   sphinx-sitemap
     -   sphinx-autobuild
+    -   sphinxcontrib.youtube
```

The Python module that you import as part of the Sphinx configuration is using the Python dotted import notation `sphinxcontrib.youtube`, but that project to install that contains it is called [sphinxcontrib-youtube](https://pypi.org/project/sphinxcontrib-youtube/).

```suggestion
    -   sphinxcontrib-youtube
```

---

@idanenglander You can use the "Commit suggestion" feature above to apply the suggestion if you'd like, or do the change manually, whatever works for you best.

---

So it seems to be complaining about that when I try building with sphinxcontrib-youtube... but not when I use the period.

`Could not import extension sphinxcontrib-youtube (exception: No module named 'sphinxcontrib-youtube')
make: *** [html] Error 2`

---

It should be `sphinxcontrib-youtube` in environment.yml and `sphinxcontrib.youtube` in conf.py, then this error should go away.

---

huzzah! You are correct. Thank you